modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/ProductPerfomance/index.tsx

```diff
@@ -453,6 +482,7 @@ const ProductPerformance = () => {
 							displayType="unstyled"
 							onClick={() => {
 								if (!products.every(isFilterAllActive)) {
+									setProductsValues('All');
```

Comment:
It's a good practice to use a variable for this like:
```suggestion
					const inicialProductValue= 'All';
					setProductsValues(inicialProductValue);
```

Reply:
Done